README.md

```diff
@@ -120,16 +111,16 @@ GitHub.authorize_url!
 Capture the `code` in your callback route on your server and use it to obtain an access token.
 
 ```elixir
-token = GitHub.get_token!(code: code)
+client = GitHub.get_token!(code: code)
```

Comment:
Hmm, this dance seems a bit much, but right now we're just using `get_token!/3` for API auth.